tg/models.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, tg: Tdlib) -> None:
         self.users = UserModel(tg)
         self.current_chat = 0
         self.downloads: Dict[int, Tuple[int, int]] = {}
+        self.selected: Dict[int, List[int]] = defaultdict(list)
+        self.yanked_msgs: Tuple[Optional[int], List[int]] = (None, [])
```

Comment:
why do you need `Optional[int]`? it will never use None.
just initialize empty tuple with `self.yanked_msgs: Tuple[int, List[int]]` and enjoy `int` type